packages/expo-image/ios/EXImageView.h

```diff
@@ -23,4 +24,52 @@
 
 - (void)didSetProps:(NSArray<NSString *> *)changedProps;
 
+@property (nonatomic, assign) UIUserInterfaceLayoutDirection reactLayoutDirection;
```

Comment:
Let's keep properties above the methods.

Reply:
Yep good one, will do 👍